### ISSUE TITLE
After verification with passphrase, I see the verify other sessions prompt but all sessions are verified

### ISSUE THREAD

**Issue by BillCarsonFr** [vector-im/element-android#4931]
### Steps to reproduce

- Login in a new session on android
- Click verify top alert
- Verify with passphrase
- wait...

=> The popup appears under the bottomsheet

<img width="400" alt="image" src="https://user-images.githubusercontent.com/9841565/149182553-cfb6d83c-406e-4521-9f77-7ae025592fa8.png">

#### What happened instead?

The verify other sessions is not needed and confusing

### Will you send logs?

No

**Comment by BillCarsonFr** [comment]
Also after verification was succesfull it looks like the request sent to my other device was not cancelled
<img width="417" alt="image" src="https://user-images.githubusercontent.com/9841565/149183082-fac62526-05ec-408b-9474-6404b634325c.png">

**Comment by BillCarsonFr** [comment]
I tried to fullfill this request and the other one is accepting
<img width="388" alt="image" src="https://user-images.githubusercontent.com/9841565/149183242-0901d18b-516f-490e-a6a5-11078fe0a73c.png">

^problem with accent colors

Not sure why it's processing the request though... Both sessions can see that it's already verified, what would happen if i click they don't match here?

**Comment by BillCarsonFr** [comment]
Similar case when verifying with other device
The `new login, was it you` appears in background while I am currently verifying it
<img width="399" alt="image" src="https://user-images.githubusercontent.com/9841565/149183896-f7ee4476-9aa5-4d20-8308-d27b6df7d2cc.png">

**Comment by ericdecanini** [comment]
I've tested 2 cases here:
1) "Verify others" prompt appears immediately after a successful verification
2) "Verification request" prompt does NOT disappear when another device starts verification process

In both cases, I found normal behaviour. Screen record attached that should show both cases (device performing verification is physical device not shown)

Perhaps this was fixed in more recent updates? Are you still able to reproduce this @BillCarsonFr?

https://user-images.githubusercontent.com/20701752/175931862-ac1bf656-ed83-43e2-946c-486dde50d2c3.mp4

**Comment by ericdecanini** [comment]
> It depends on how fast is the Home server, and the size of your account. We are missing some sort of `trust loading` state, currently it's trusted|untrusted. We probably want a 3rd state `computing`

This seems to hold true. Once tested on a bigger account I was able to reproduce the issue (on matrix.org homeserver)

**Comment by kittykat** [comment]
@ericdecanini @BillCarsonFr could this be a duplicate of #4318

**Comment by pc00per** [comment]
This issue seems so severe lmao.

**Comment by kittykat** [comment]
> This issue seems so severe lmao.

Yes, we agree that it is. This is why we are triaging and assigning related issues so that there is one source of truth for the work and it's easier to track & action :+1:
